regional_mom6/regional_mom6.py

```diff
@@ -440,7 +440,7 @@ def rectangular_hgrid(λ, φ):
     dλ = λ[1] - λ[0]
 
     assert (
-        np.max(np.diff(λ) - dλ) < 1e-14
+        np.max(np.diff(λ) - dλ) < 1e-10
```

Comment:
The function `numpy.allclose` takes into account numerical tolerances better.

```suggestion
        np.allclose(np.diff(λ), dλ * np.ones(np.size(λ)-1))
```

Indeed the snippet you posted was failing for me also. The code I suggest above works for even down to `resolution = 0.000001`...!